aiken-contracts/fractional/hashes/cip68.hash

```diff
@@ -1 +1 @@
-7d5b23ebea44f43508c55dd9a25c554991c5de48247922c07bfb4c8c
+1a49d7da705362d34b8ca72cd78306747be234055bf57e51404d8d9f
```

Comment:
Any files that change when something is tweaked and re-compiled can be removed from the repository entirely. As long as they are outputs of the build process, they don't need to be in github.

Reply:
Alright Ill ignore all the folders that are filled with stuff that is compile dependent.